examples/helm-deployment/render.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This file seems an odd addition for this example.  Can we add it to `integration/testdata/helm` instead?

Reply:
🤦 this isn't supposed to be here. removed